M2Mqtt/Net/MqttNetworkChannel.cs

```diff
@@ -230,6 +236,17 @@ public MqttNetworkChannel(string remoteHostName, int remotePort, bool secure, X5
 #endif
         }
 
+#if !(MF_FRAMEWORK_VERSION_V4_2 || MF_FRAMEWORK_VERSION_V4_3 || COMPACT_FRAMEWORK)
+        /// <param name="userCertificateSelectionCallback">A RemoteCertificateValidationCallback delegate responsible for validating the certificate supplied by the remote party</param>
+        /// <param name="userCertificateValidationCallback">A LocalCertificateSelectionCallback delegate responsible for selecting the certificate used for authentication</param>
+        public MqttNetworkChannel(string remoteHostName, int remotePort, bool secure, X509Certificate caCert, X509Certificate clientCert, MqttSslProtocols sslProtocol,
+            RemoteCertificateValidationCallback userCertificateValidationCallback,
+            LocalCertificateSelectionCallback userCertificateSelectionCallback, List<string> alpnProtocols)
```

Comment:
Please make the parameter optional or introduce an overload. And align all usages with `null` for this param.

Reply:
I'm very sorry I'm not sure what you mean in this comment.
This constructor which I add is added as new another one, so it's already an overload constructor.

Some MqttNetworkChannel constructor existing  already, so I thought I should not change exsisting constructors and I added another.

You mean, I should set to parameter "aplnProtocols" a default value null?
`...
userCertificateSelectionCallback, List<string> alpnProtocols = null)
`
Please tell me more detail you mean.
(It's very helpful for me if you show some code snippet or something)

Reply:
Please make your constructor default one and previous one calling this one. I think this would be cleaner and all code concentrated on one place. Or remove yours and add optional parameter to the default one.